cfgov/jinja2/v1/_includes/molecules/hero.html

```diff
@@ -34,31 +34,65 @@
 
    ========================================================================== #}
 
-{% macro render( settings, options={} ) %}
-
-{% if settings.heading and settings.body and settings.image_url %}
+{% if value %}
 <section class="hero"
          {{ 'style=background-color:' +
-             options.background_color + ';'
-             if options.background_color else '' }}>
+             value.background_color + ';'
+             if value.background_color else '' }}>
     <div class="wrapper hero_wrapper">
         <div class="hero_text">
-            <h1 class="hero_heading">{{ settings.heading }}</h1>
-            <p class="hero_subhead">{{ settings.body }}</p>
-            {% if options.link_text and options.link_url %}
-                <a href="{{ options.link_url }}"
+            <h1 class="hero_heading">{{ value.heading | safe}}</h1>
+            <p class="hero_subhead">{{ value.body |richtext}}</p>
+            {% if value.link.text and value.link.url %}
+                <a href="{{ value.link.url }}"
                    class="hero_cta
-                   {{ 'btn' if options.is_button
+                   {{ 'btn' if value.is_button
                    else '' }}">
-                    {{ options.link_text }}
+                    {{ value.link.text }}
                 </a>
             {% endif %}
         </div>
-        <div class="hero_image"
-             style="background-image: url( {{ settings.image_url }} );">
+        {% if value.image.upload %}
+            {% set photo=image(value.image.upload, 'original') %}
+            <div class="hero_image"
+                 style="background-image: url( {{ photo.url }} );">
+        {% else %}
```

Comment:
When we talked with content earlier last week, they said they would just want the image upload and not the image url option. Since it isn't clear that you have to put in one or the other until you submit, can we remove the second option? @sarahsimpson09 

![image](https://cloud.githubusercontent.com/assets/1860176/11283981/64acf9fe-8ed5-11e5-9de2-1cc8b18e8888.png)